include/boost/archive/codecvt_null.hpp

```diff
@@ -65,7 +65,7 @@ class codecvt_null<char> : public std::codecvt<char, char, std::mbstate_t>
 template<>
 class BOOST_SYMBOL_VISIBLE codecvt_null<wchar_t> : public std::codecvt<wchar_t, char, std::mbstate_t>
 {
```

Comment:
Below: In both cases BOOST_WARCHIVE_DECL is already setting export, so this resolves a warning.